src/components/event-components/EventTile.tsx

```diff
@@ -2,117 +2,90 @@ import { EventFull } from '~/lib/sanity.types'
 import React, { FC, useState } from 'react'
 import { EventCoverPicture } from '~/components/event-components/EventCoverPicture'
 import { CalendarIcon } from '~/components/svg-components/CalendarIcon'
-import { PictureIcon } from '~/components/svg-components/PictureIcon'
-import { DocumentIcon } from '~/components/svg-components/DocumentIcon'
-import { IconBox } from '~/components/event-components/IconBox'
-import { CalendarAndClockIcon } from '~/components/svg-components/CalendarAndClockIcon'
 import { LocationIcon } from '~/components/svg-components/LocationIcon'
 import { LecturerIcon } from '~/components/svg-components/LecturerIcon'
-import { ChevronUpIcon } from '@heroicons/react/24/solid'
-import { DateTime } from 'groq-js'
+import { formatDateTime } from '~/utils/format-date-time'
+import { PictureIcon } from '~/components/svg-components/PictureIcon'
+import { useRouter } from 'next/router'
 
 interface Props {
-  eventSummary: EventFull
+  event: EventFull
 }
 
-export const EventTile: FC<Props> = ({ eventSummary }) => {
+export const EventTile: FC<Props> = ({ event }) => {
   const [hovered, setHovered] = useState<boolean>(false)
-
-  function formatDateTime(datetime: DateTime): String {
-    return datetime
-      .toString()
-      .substring(0, 16)
-      .replace('T', ' ')
-      .replaceAll('-', '.')
-  }
+  const router = useRouter()
+  const iconSize = 25
 
   return (
     <div
-      className={`relative transition-all bg-gray-900  rounded-md max-w-2xl overflow-y-hidden ${hovered ? 'md:scale-105 ' : ''} `}
+      className="relative transition-transform bg-gray-900 rounded-md max-w-2xl overflow-hidden cursor-pointer"
       onMouseEnter={() => setHovered(true)}
       onMouseLeave={() => setHovered(false)}
     >
-      {eventSummary.image && (
-        <div
-          className={`transition-all relative z-10 ${hovered ? '-translate-y-[105%]' : ''}`}
-        >
-          <EventCoverPicture
-            image={eventSummary.image}
-            title={eventSummary.title}
-          />
-          <div className="absolute bottom-0 right-0 left-0 w-full">
-            <ChevronUpIcon className="w-10 mx-auto" />
+      {event.image && (
+        <div className="relative h-full">
+          <div
+            className={`transition-transform ${
+              hovered ? '-translate-y-[40%]' : ''
+            }`}
+          >
+            <a
+              onClick={() => {
+                const path =
+                  event.externalLink || '/event/' + event.slug.current
+                router.push(path)
+              }}
+            >
```

Comment:
Right now, you only get taken to the event page if the image is clicked. I would move to outer div so the whole box is clickable